demo/intel-gpu-tensorflow/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+  - configmap.yaml
```

Comment:
use `configMapGenerator`. See https://github.com/intel/intel-device-plugins-for-kubernetes/blob/eeef8aa354ef1fa0ea35ac8e3d4edb6993eedcb5/deployments/sgx_aesmd/base/kustomization.yaml#L5

Reply:
Thanks, with that I can have the .py file as is.